Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -39,9 +44,28 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def find_conflict_pr(cls, remote, branch):
-        for pr in remote.pull_requests.find(head=branch, opened=True):
-            return pr
+    def find_conflict_pr(cls, remote, radar_id):
+
+        radar = Tracker().client.radar_for_id(radar_id, additional_fields=['sourceChanges'])
+        shas = []
+
+        for entry in radar.sourceChanges.splitlines():
+            repo, action, sha = entry.split(', ')
+            if repo == 'WebKit':
```

Comment:
To stay generic the repo name may need to be calculated from git information. Other places use `repo_name = rmt.name if '/' not in name else name.split('/', 1)[-1]`

Reply:
messaged cam offline, I think we want hardcoded webkit here since that's what we put in the sourceChanges field.